pandas/core/frame.py

```diff
@@ -4554,13 +4554,14 @@ def unstack(self, level=-1, fill_value=None):
     id_vars : tuple, list, or ndarray, optional
         Column(s) to use as identifier variables.
     value_vars : tuple, list, or ndarray, optional
-        Column(s) to unpivot. If not specified, uses all columns that
-        are not set as `id_vars`.
-    var_name : scalar
-        Name to use for the 'variable' column. If None it uses
+        Column(s) to unpivot. If list of lists, simultaneously unpivot
+        each sublist into its own variable column. If not specified, uses all
```

Comment:
add a mention where this changes as appropriate (meaning in 0.22.0) in the doc-string